tests/migrations/test_writer.py

```diff
@@ -613,16 +613,21 @@ def test_migration_file_header_comments(self):
         })
         dt = datetime.datetime(2015, 7, 31, 4, 40, 0, 0, tzinfo=utc)
         with mock.patch('django.db.migrations.writer.now', lambda: dt):
-            writer = MigrationWriter(migration)
-            output = writer.as_string()
-
-        self.assertTrue(
-            output.startswith(
-                "# Generated by Django %(version)s on 2015-07-31 04:40\n" % {
-                    'version': get_version(),
-                }
-            )
-        )
+            for include_header in (True, False):
+                with self.subTest("Test migration file header comments, include_header=%s" % include_header):
```

Comment:
We don't often use the `msg` param with `subTest()`. 

Maybe: 

`"Migration file includes header: %s." % include_header`

or just: 

`include_header=include_header`

Reply:
Done